monocle/github/pullrequest.py

```diff
@@ -213,6 +214,9 @@ def _getPage(self, kwargs, prs):
             self.log.info("Total PRs to fetch: %s" % kwargs['total_prs_count'])
         if 'data' not in data:
             self.log.error('No data collected: %s' % data)
+            if 'message' in data and 'wait a few minutes' in data['message']:
+                self.log.debug('sleeping 120s')
```

Comment:
Let's give a bit more information in the log message. I would have set it to info.

Reply:
or warning

Reply:
ok